handler/src/main/java/io/netty/handler/ssl/OpenSslSessionContext.java

```diff
@@ -62,9 +63,21 @@ public SSLSession getSession(byte[] bytes) {
      */
     @Deprecated
     public void setTicketKeys(byte[] keys) {
-        ObjectUtil.checkNotNull(keys, "keys");
+        if (ObjectUtil.checkNotNull(keys, "keys").length % SessionTicketKey.TICKET_KEY_SIZE != 0) {
```

Comment:
`ObjectUtil.checkNotNull(keys, "keys")` -> `keys`. A NPE will be thrown anyways and `keys` is the only paramter.